src/Driver/Driver.jl

```diff
@@ -224,6 +225,12 @@ function parse_commandline(
         metavar = "<level>"
         arg_type = String
         default = uppercase(get_setting(:log_level, defaults, global_defaults))
+        "--custom-logger"
```

Comment:
```  
 "--disable-custom-logger"
        help = "Disable using ClimateMachine's custom MPI aware logger"
        action = :store_true
```

Reply:
?

Reply:
Why `--disable-custom-logger`? And what does `action` do?

Reply:
it's consistent with the other binary flag options for the cli.  

action :store_true just sets the arg variable to true if the flag is present, see `disable-gpu`

Reply:
Shouldn't the help match `custom_logger = true` indicating that we're using CM.jl's custom logger?

Reply:
@simonbyrne preferred this way (I do too, to avoid the double negative `--disable-custom-logger=false`). Let's work out the details in another PR (I'd like to continue with the uphill docs battle!).

Reply:
sure, no problem.  just pointing out that it's not a double negative when it's a binarly cli flag.  for flags there is not "=true" or "=false", if it exists it is marked true.